pgctl/cli.py

```diff
@@ -53,8 +53,22 @@ class NoSuchService(Exception):
 def svc(*args):
     p = Popen(('svc',) + tuple(args), stdout=PIPE, stderr=PIPE)
     _, stderr = p.communicate()
-    if 'file does not exist' in stderr:
+    if 'unable to chdir' in stderr:
         raise NoSuchService(stderr)
+    return p.returncode
+
+
+def stat(*args):
```

Comment:
You might improve this to cover the case you have in test:

Return the state of the service as a string.
None representing unknown service?